Graphics/GraphicsEngineOpenGL/src/GLContextAndroid.cpp

```diff
@@ -142,6 +152,19 @@ bool GLContext::InitEGLSurface()
 
 bool GLContext::InitEGLContext()
 {
+    if (context_non_owned_)
+    {
+        glGetIntegerv(GL_MAJOR_VERSION, &major_version_);
+        glGetIntegerv(GL_MINOR_VERSION, &minor_version_);
+        context_ = eglGetCurrentContext();
+        if (context_ == EGL_NO_CONTEXT)
+        {
+            LOG_ERROR_AND_THROW("Failed to attach to existing EGL context");
+        }
+        context_valid_ = true;
+        return true;
```

Comment:
There is `AttachToCurrentEGLContext()` function that does the same

Reply:
It doesn't or its mislabeled. It assumes that you are checking for the presence of the one created by diliegent engine. It needs to be reworked in my opinion if that is the intention.